### ISSUE TITLE
Fail to install on Arch Linux.

### ISSUE THREAD

**Issue by playXE** [zigtools/zls#99]
Running with `zig build config` and getting these errors:
```
error: FileNotFound
/usr/lib/zig/std/os.zig:1767:19: 0x274e8c in std.os.mkdiratZ (build)
        EEXIST => return error.PathAlreadyExists,
                  ^
/usr/lib/zig/std/os.zig:1750:9: 0x274d5b in std.os.mkdirat (build)
        return mkdiratZ(dir_fd, &sub_dir_path_c, mode);
        ^
/usr/lib/zig/std/fs.zig:744:9: 0x274bf6 in std.fs.Dir.makeDir (build)
        try os.mkdirat(self.fd, sub_path, default_new_dir_mode);
        ^
/usr/lib/zig/std/os.zig:930:23: 0x25e40b in std.os.openatZ (build)
            ENOENT => return error.FileNotFound,
                      ^
/usr/lib/zig/std/fs.zig:864:25: 0x297c45 in std.fs.Dir.openDirFlagsZ (build)
            else => |e| return e,
                        ^
/usr/lib/zig/std/fs.zig:835:13: 0x2886e8 in std.fs.Dir.openDirZ (build)
            return self.openDirFlagsZ(sub_path_c, os.O_DIRECTORY | os.O_RDONLY | os.O_CLOEXEC | O_PATH);
            ^
/usr/lib/zig/std/fs.zig:822:13: 0x27b40f in std.fs.Dir.openDir (build)
            return self.openDirZ(&sub_path_c, args);
            ^
/home/aprokurov/zls/build.zig:19:15: 0x27e7ee in std.special.config (build)
    var dir = try std.fs.cwd().openDir(builder.exe_dir, .{});
              ^
/usr/lib/zig/std/build.zig:2394:9: 0x275cbe in std.build.Step.make (build)
        try self.makeFn(self);
        ^
/usr/lib/zig/std/build.zig:404:9: 0x2746b3 in std.build.Builder.makeOneStep (build)
        try s.make();
        ^
/usr/lib/zig/std/build.zig:359:13: 0x26a9e7 in std.build.Builder.make (build)
            try self.makeOneStep(s);
            ^
/usr/lib/zig/std/special/build_runner.zig:125:21: 0x26339d in main (build)
            else => return err,
                    ^

Build failed. The following command failed:
/home/aprokurov/zls/zig-cache/o/xvCpi0WL6uGJa0YULTtO8TZpjpETQ0KcZo-_-aUr3ilWRLYVOwYH3Plxkbapz707/build /usr/bin/zig /home/aprokurov/zls /home/aprokurov/zls/zig-cache config

```

**Comment by playXE** [comment]
Installed Zig using `pacman -S zig`,std is in /usr/lib/zig/std

**Comment by SuperAuguste** [comment]
Hey @playXE! Thanks for reporting this issue! Did you run `zig build` first?

**Comment by playXE** [comment]
@SuperAuguste yes, It gives different error 
```
Semantic Analysis [721/1154] ./src/document_store.zig:16:18: error: container 'std' has no member called 'ArrayListUnmanaged'
    packages: std.ArrayListUnmanaged(Pkg),
                 ^
./src/document_store.zig:25:5: note: while checking this field
    associated_build_file: ?*BuildFile,
    ^
/usr/lib/zig/std/hash_map.zig:52:13: note: while checking this field
            value: V,
            ^
/usr/lib/zig/std/hash_map.zig:58:13: note: while checking this field
            kv: KV,
            ^
/usr/lib/zig/std/hash_map.zig:35:9: note: while checking this field
        entries: []Entry,
        ^
./src/document_store.zig:76:1: note: while checking this field
handles: std.StringHashMap(*Handle),
^
./src/main.zig:18:37: note: referenced here
var document_store: DocumentStore = undefined;
                                    ^
./src/main.zig:986:13: note: referenced here
        try document_store.init(allocator, zig_exe_path, try std.mem.dupe(allocator, u8, build_runner_path));
            ^
zls...The following command exited with error code 1:
/usr/bin/zig build-exe /home/aprokurov/zls/src/main.zig --pkg-begin build_options /home/aprokurov/zls/zig-cache/zls_build_options.zig --pkg-end --cache-dir /home/aprokurov/zls/zig-cache --name zls --pkg-begin known-folders /home/aprokurov/zls/src/known-folders/known-folders.zig --pkg-end --cache on 

Build failed. The following command failed:
/home/aprokurov/zls/zig-cache/o/k3-01oLj_34RMNZfRlUp1DH1UFF8FcLIXLlUvKCFqCsyEnl1n0aYAa-bEv_3M_g4/build /usr/bin/zig /home/aprokurov/zls /home/aprokurov/zls/zig-cache
```

**Comment by data-man** [comment]
@playXE 
> Installed Zig using pacman -S zig

From readme:
>Note: **zls itself must be built using the master branch of zig** currently due to a bug in std.json which was fixed after 0.6.0 was released, as well as various parser changes that have happened since.

**Comment by playXE** [comment]
Oh, I didn't read this, thanks for help!
